databases/week3/create_meals_tables.sql

```diff
@@ -0,0 +1,41 @@
+-- Data model
+
+create database meals;
+use meals;
+
+create table meal (
+	  id int(10) unsigned NOT NULL AUTO_INCREMENT,
+	  title varchar(255) NOT NULL,
+	  description text NULL default NULL,
+	  location varchar(255) NULL, 
+	  theday datetime NOT NULL, -- it was showing error if I use "When"
+	  max_reservations int(10) unsigned NOT NULL,
+	  price decimal NOT NULL,
```

Comment:
Great that you are using a decimal for storing the price! :) One thing to note is that without any explicit arguments it will default to `DECIMAL(10, 0)` which means you can not store anything after the point fx `99.95`.

I would recommend using `DECIMAL(13, 4)` when dealing with money. See: https://rietta.com/blog/best-data-types-for-currencymoney-in/

Reply:
Ok got it. I made the changes.